debezium-connector-jdbc/src/main/java/io/debezium/connector/jdbc/naming/CustomCollectionNamingStrategy.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.jdbc.naming;
+
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.connector.jdbc.util.NamingStyle;
+import io.debezium.connector.jdbc.util.NamingStyleUtils;
+import io.debezium.sink.DebeziumSinkRecord;
+import io.debezium.sink.naming.CollectionNamingStrategy;
+
+/**
+ * Custom implementation of the {@link CollectionNamingStrategy} interface that allows
+ * transforming collection names using various naming styles and optional prefix/suffix.
+ * <p>
+ * This strategy can extract the table name from a topic name of format:
+ * <ul>
+ *   <li>server.schema.table</li>
+ *   <li>schema.table</li>
+ *   <li>table</li>
+ * </ul>
+ * <p>
+ * The table name is then transformed using the configured naming style, and
+ * prefix/suffix can be applied to create the final table name.
+ *
+ * @author Gustavo Lira
+ */
+public class CustomCollectionNamingStrategy implements CollectionNamingStrategy {
```

Comment:
I believe this class and any references/tests can be removed as its not used any longer.